pkg/ruptela/ruptela.go

```diff
@@ -57,3 +57,13 @@ func unplugged(originalDoc []byte) bool {
 func ConvertPSIToKPa(psi float64) float64 {
 	return psi * 6.89476
 }
+
+func ignoreOutOfHeadingRange(val float64, err error) (float64, error) {
```

Comment:
I don't think we need a new function for this since it will only be used for the  heading conversion instead it could just live in the `ToCurrentLocationHeading0` function